api/swagger/swagger.yaml

```diff
@@ -42,6 +41,70 @@ paths:
           description: Error
           schema:
             $ref: "#/definitions/ErrorResponse"
+  /:userId/roster:
+    x-swagger-router-controller: roster
+      get:
+        description: Returns the user's complete roster of teams
+        operationId:
+        responses:
+      post:
+        description: Creates a new team for the user's roster
+        operationId:
+        parameters:
+          - name: team
+            in: query
+            description: The name of the team to create
+            required: true
+            type: string
+        responses:
+      delete:
+        description: Deletes the user's specified teams
+        operationId:
+        parameters:
+          - name: team
+            in: query
+            description: The name of the team to delete
+            required: true
+            type: string
+        responses:
+  /:userId/roster/:team:
+    x-swagger-router-controller: roster
+      get:
+        description: Returns the user's specific team
+        operationId:
+        responses:
+  /:userId/roster/:team/:position:
+    x-swagger-router-controller: roster
+      post:
+        description: Adds one player to the user's team at the specified position
+        operationId:
+        parameters:
+          - name: player
+            in: query
+            description: The player to add to the team
+            required: true
+            type: # TBD As a user, do I input the playerID or player name?
```

Comment:
I think if we can make the player name their unique id then that saves us a lot of work. If we do player id then their id's will be set based on what position a player was added to the DB. If so, how do we retrieve that player if all we're using is their unique id?